### PR TITLE
Created post-card route + view

### PR DESCRIPTION
**Pull request by EthanJStark**

### CODE REVIEW

**Comment by EthanJStark** [comment]
@jrob8577, please review as you're able. Thanks!
bitmoji

File: database/db.js [hunk 1]
```diff
@@ -6,11 +6,17 @@ const getAllDecks = 'SELECT * FROM decks'
...
-const Decks = {
+const newCard = 'INSERT INTO cards (front, back, deck_id) VALUES ($1, $2, $3)'
+
+const Deck = {
   all: () => db.any( getAllDecks ),
   new: title => db.one( newDeck, [title] )
 }
 
+const Card = {
+  new: (front, back, id) => db.one( newCard, [front, back, id] )
```
Comment (jrob8577) [inline database/db.js]: `id` is a misleading name here - maybe `deck_id`?

Comment (jrob8577) [inline database/db.js]: `id` is a misleading name here - maybe `deck_id`?

Comment (jrob8577) [inline database/db.js]: `id` is a misleading name here - maybe `deck_id`?

**Pull request merged**